### PR TITLE
Train compressor script

### PR DESCRIPTION
**Pull request by Justinezgh**
Hi @dlanzieri! [Here](https://github.com/DifferentiableUniverseInitiative/sbi_lens/blob/u/Justinezgh/compressor_new_sim/scripts/train_compressor.py) you can find the script to train the simulator. If everything works properly, you just have to change `--loss` to `'train_compressor_mse'` to train the compressor using mse (but have not tried it yet ^^). 

To train the compressor using vmim, I used the following configuration:
`--total_steps = 150_000  --resnet = 'resnet18'  --loss = 'train_compressor_vmim'`

I also add [here](https://github.com/DifferentiableUniverseInitiative/sbi_lens/blob/u/Justinezgh/compressor_new_sim/sbi_lens/gen_dataset/utils.py) the augmentation function (to add the noise on the fly etc) and tested it in [this notebook](https://gist.github.com/Justinezgh/fb42e60f3619741be3c8f4af33f2b7bd).

[Here](https://github.com/DifferentiableUniverseInitiative/sbi_lens/blob/u/Justinezgh/compressor_new_sim/sbi_lens/config.py) I created the lsst year 10 configuration with the fiducial etc (just to avoid mistakes). 

And in the [data](https://github.com/DifferentiableUniverseInitiative/sbi_lens/tree/u/Justinezgh/compressor_new_sim/sbi_lens/data) folder I updated the m_data observation and the power spectrum chains + I added the chains from the full field mcmc.

### CODE REVIEW

**Comment by EiffL** [comment]
@dlanzieri @Justinezgh let's not forget to review and merge this branch from 2 weeks ago :-)

**Comment by dlanzieri** [comment]
@Justinezgh Maybe is not the right branch and PR, but we should also update the [utils.py](sbi_lens/simulator/utils.py), with the default values we used to get the full field contours and the same strategy (e.g. `init_strategy=partial(numpyro.infer.init_to_value, values=init_values)` instead of `init_strategy=numpyro.infer.init_to_median`)

**Comment by Justinezgh** [comment]
Hi @dlanzieri yes sure it was on my todo for today :) I will open a new pr and add the sequential mcmc that I used for the full field

**Review by dlanzieri — APPROVED** [review]
